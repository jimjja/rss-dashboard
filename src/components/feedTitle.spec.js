import React from 'react';
import renderer from 'react-test-renderer';
import FeedTitle from './feedTitle';

describe('Feed Title component', () => {
  it('should match snapshot', () => {
    const props = {
      title: 'Example title',
    };
    const tree = renderer.create(<FeedTitle {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
