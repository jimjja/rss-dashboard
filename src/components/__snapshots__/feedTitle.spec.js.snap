// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feed Title component should match snapshot 1`] = `
<div
  style={
    Object {
      "border": "1px solid #ebedf0",
      "borderLeft": 0,
      "borderRight": 0,
      "borderTop": 0,
    }
  }
>
  <h2>
    Example title
  </h2>
</div>
`;
